.travis.yml

```diff
@@ -21,7 +21,7 @@ install:
 
 script:
   - make test
-  - make xlang
+  - make xlang-fresh
```

Comment:
It looks like this might be related to `docker-compose run xlang --rm` not stopping and removing its dependencies. The `hello-server` is running from previous failed `docker-compose run xlang --rm` runs.

https://github.com/docker/compose/issues/2791